src/util/certs/create-cert-for-cns.ts

```diff
@@ -64,6 +64,20 @@ export default async function createCertForCns(
     if (error.code === 'wildcard_not_allowed') {
       return new ERRORS.WildcardNotAllowed(error.domain);
     }
+    if (error.code === 'unauthorized_request_error') {
+      return new ERRORS.UnauthorizedCertsRequestError(
+        error.response.detail,
+        error.response.type,
+        error.response.domain
+      );
+    }
+    if (error.code === 'dns_error') {
+      return new ERRORS.CertsDNSError(
+        error.response.detail,
+        error.response.type,
+        error.response.domain
```

Comment:
@anatrajkovska replace `domain: string` with `cns: string[]` and pass `cns` value from original function argument. We don't return `domain` in the api error response.